draft-ietf-tls-esni.md

```diff
@@ -353,15 +403,17 @@ ClientHello. This extension is defined as follows:
 ~~~~
 
 nonce
-: A 16-byte nonce exported from the HPKE encryption context. See {{send-ech}}
-for details about its computation.
-
-Finally, requirements in {{client-behavior}} and {{server-behavior}} require
-implementations to track, alongside each PSK established by a previous
-connection, whether the connection negotiated this extension with the
-"ech_accept" response type. If so, this is referred to as an "ECH PSK".
-Otherwise, it is a "non-ECH PSK". This may be implemented by adding a new field
-to client and server session states.
+: A 16-byte nonce exported from the HPKE encryption context, as described in
+{{send-ech}}.
+
+Per {{client-behavior}} and {{server-behavior}}, implementations are required to
+track, alongside each PSK established by a previous connection, whether the
+connection negotiated this extension with the "ech_accept" response type. If so,
+this is referred to as an "ECH PSK". Otherwise, it is a "non-ECH PSK". This may
+be implemented by adding a new field to client and server session states.
+[[TODO: Why does discussion of the PSK belong in this section, which is about
+"ech_nonce"?]]
+[[TODO: The "'ech_accept' response type" hasn't been defined here.]]
```

Comment:
Good point. We should probably shuffle things around.

Reply:
Let's either remove the TODO and file an issue, or address this here.

Reply:
Actually, this is the *only* place in the document where "ech_accept" appears. This needs to be defined.

Reply:
Is this an issue, or shall we define the alert here? I'm not sure how to do this.

Reply:
Let's just make it an issue. It'll be a small change.

Reply:
Done. https://github.com/tlswg/draft-ietf-tls-esni/issues/267